diy-omd.ino

```diff
@@ -117,15 +174,33 @@ void setup() {
   loadParameters();
 
   pinMode(hallEffectSensorPin, INPUT);
+
+  delay(3000);
   ESC1.attach(escPin1, 1000, 2000); // (pin, min pulse width, max pulse width in microseconds) 
   ESC2.attach(escPin2, 1000, 2000); // (pin, min pulse width, max pulse width in microseconds) 
-  ESC3.attach(escPin3, 1000, 2000); // (pin, min pulse width, max pulse width in microseconds) 
-  delay(1000);
+  ESC3.attach(escPin3, 1000, 2000); // (pin, min pulse width, max pulse width in microseconds)
+
+  if (ESC_TYPE == SIDEWINDER){
+    Serial.println("Ready Sidewinder ESC");
+//    delay(3000);
+//    ESC1.write(180);
+//    ESC2.write(180);
+//    ESC3.write(180);
+//    delay(1000);
+//    ESC1.write(0);
+//    ESC2.write(0);
+//    ESC3.write(0);
+//    delay(1000);
+    ESC1.write(90);
+    ESC2.write(90);
+    ESC3.write(90);
```

Comment:
This looks like the existing INIT mode we have for the RC type ESCs. Does that not work for you?

Reply:
Yes, yours worked, I had some issues unrelated to your code and will remove this bit

Reply:
Removed